### ISSUE TITLE
invalid helm options

### ISSUE THREAD

**Issue by travisghansen** [J12934/helm-gh-pages-action#5]
I'm guessing `ubuntu-latest` now has `helm-v3` bundled?

```
helm init --client-only
Error: unknown flag: --client-only
##[error]The process 'helm' failed with exit code 1
```

**Comment by J12934** [comment]
Oh good catch.

I looked a the action virtual environment docs and its appernetly mixed.

The Ubuntu 16 page lists helm 3
The Ubuntu 18 page lists helm 2

I guess I'll just publish a new release with a notice that it will only support helm 3 from now on.

**Comment by J12934** [comment]
Should be fixed on master and `v2.0.0`. Thanks again 👍

**Comment by travisghansen** [comment]
Appears to be working, thanks!
